src/libraries/Microsoft.Extensions.Logging.Console/tests/TrimmingTests/AddConsoleFormatterTests.cs

```diff
@@ -17,8 +17,10 @@ static int Main(string[] args)
         IServiceCollection descriptors = new ServiceCollection();
         descriptors.AddLogging(builder =>
         {
+#pragma warning disable CA1416 // Unsupported on Android
```

Comment:
It should use IsAndroid check if we keep it as unsupported

Reply:
`IsAndroid` check is not available here b/c "OperatingSystem doesn't contain" it during the build. "#if TARGET_ANDROID" doesn't work as well.

Reply:
What's the error you get? The API is available since 5.0 https://docs.microsoft.com/en-us/dotnet/api/system.operatingsystem.isandroid?view=net-5.0

Reply:
It's because `Microsoft.Extensions.Logging.Console` targets `netstandard2.0;net461`

Reply:
Oh, so the test is actually not a real test app but custom build version of Microsoft.Extension.Logging.Console library? @eerhardt why are we doing it this way?

Reply:
I pushed another change, perhaps this pragma is no longer topical.

Reply:
I'm not sure I understand the question. This is a "trimming test" that Microsoft.Extensions.Logging.Console works in a trimmed app.

If Android doesn't support Console, we should just skip this test on Android. See 

https://github.com/dotnet/runtime/blob/6b64618b0b09e430a514d5b718bd46ce08f0b359/src/libraries/Microsoft.Extensions.Logging.Console/tests/TrimmingTests/Microsoft.Extensions.Logging.Console.TrimmingTests.proj#L6